metrics_utility/automation_controller_billing/collectors.py

```diff
@@ -524,3 +537,57 @@ def main_host_table(since, full_path, until, **kwargs):
     return _copy_table(
         table='main_host', query=f'COPY ({query}) TO STDOUT WITH CSV HEADER', path=full_path, prepend_query=yaml_and_json_parsing_functions()
     )
+
+
+@register('total_workers_vcpu', '1.0', format='json', description=_('Total workers vCPU'), fnc_slicing=limit_slicing)
+def total_workers_vcpu(since, full_path, until, **kwargs):
+    if 'total_workers_vcpu' not in get_optional_collectors():
+        return None
+
+    cluster_name = os.environ.get('METRICS_UTILITY_CLUSTER_NAME')
+    if not cluster_name:
+        logger.error('environment variable METRICS_UTILITY_CLUSTER_NAME is not set')
+        raise MissingRequiredEnvVar('environment variable METRICS_UTILITY_CLUSTER_NAME is not set')
+
+    now = datetime.now(timezone.utc)
+
+    info = {'cluster_name': cluster_name, 'timestamp': now.isoformat(), 'nodes': []}
+    # If METRICS_UTILITY_USAGE_BASED_BILLING_ENABLED is not set or set to false then it returns 1
+    usage_based_billing_enabled_str = os.environ.get('METRICS_UTILITY_USAGE_BASED_BILLING_ENABLED')
+    usage_based_billing_enabled = False
+    if usage_based_billing_enabled_str and (usage_based_billing_enabled_str.lower() == 'true'):
+        usage_based_billing_enabled = True
+    if not usage_based_billing_enabled:
+        return {'cluster_name': info['cluster_name'], 'total_workers_vcpu': 1}
```

Comment:
Shall we add a log entry in this scenario too for the warm fuzzy that our existing customers are behaving correctly?